lib/chat/ui/widgets/bubble_message.dart

```diff
@@ -0,0 +1,184 @@
+    this.textStyle,
+    this.textContainerMargin,
+    this.rowMainAxisAlignment,
+    this.rowCrossAxisAlignment,
+  });
+
+  MessageBubbleStyle copyWith({
+    EdgeInsetsGeometry outerContainerMargin,
+    BorderRadiusGeometry textContainerBorderRadius,
+    EdgeInsetsGeometry textContainerPadding,
+    EdgeInsetsGeometry textContainerMargin,
+    Color textContainerBackgroundColor,
+    TextAlign textAlignment,
+    TextStyle textStyle,
+    CrossAxisAlignment rowCrossAxisAlignment,
+    MainAxisAlignment rowMainAxisAlignment,
+  }) {
+    return MessageBubbleStyle(
+      textContainerBorderRadius:
+          textContainerBorderRadius ?? this.textContainerBorderRadius,
+      textAlignment: textAlignment ?? this.textAlignment,
+      textStyle: textStyle ?? this.textStyle,
+      textContainerBackgroundColor:
+          textContainerBackgroundColor ?? this.textContainerBackgroundColor,
+      rowCrossAxisAlignment:
+          rowCrossAxisAlignment ?? this.rowCrossAxisAlignment,
+      textContainerMargin: textContainerMargin ?? this.textContainerMargin,
+      outerContainerMargin: outerContainerMargin ?? this.outerContainerMargin,
+      textContainerPadding: textContainerPadding ?? this.textContainerPadding,
+      rowMainAxisAlignment: rowMainAxisAlignment ?? this.rowMainAxisAlignment,
+    );
+  }
+}
+
+class _DefaultMessageBubbleStyle extends MessageBubbleStyle {
+  final EdgeInsetsGeometry outerContainerMargin = const EdgeInsets.symmetric(
+    vertical: 10.0,
+    horizontal: 20.0,
+  );
+  final BorderRadiusGeometry textContainerBorderRadius =
+      const BorderRadius.all(Radius.circular(20.0));
+  final EdgeInsetsGeometry textContainerPadding = const EdgeInsets.all(16.0);
+  final EdgeInsetsGeometry textContainerMargin = const EdgeInsets.only(
+    left: 30.0,
+    right: 0.0,
+  );
+  final Color textContainerBackgroundColor = const Color(0xFF00CD9F);
+  final TextAlign textAlignment = TextAlign.start;
+  final TextStyle textStyle = const TextStyle(
+    color: const Color(0xFFFFFFFF),
+    fontSize: 15.0,
+  );
+  final CrossAxisAlignment rowCrossAxisAlignment = CrossAxisAlignment.end;
+  final MainAxisAlignment rowMainAxisAlignment = MainAxisAlignment.end;
+
+  const _DefaultMessageBubbleStyle({
+    EdgeInsetsGeometry outerContainerMargin,
+    BorderRadiusGeometry textContainerBorderRadius,
+    EdgeInsetsGeometry textContainerPadding,
+    EdgeInsetsGeometry textContainerMargin,
+    Color textContainerBackgroundColor,
+    TextAlign textAlignment,
+    TextStyle textStyle,
+    CrossAxisAlignment rowCrossAxisAlignment,
+    MainAxisAlignment rowMainAxisAlignment,
+  });
+}
+
+const MessageBubbleStyle _defaultStyle = const _DefaultMessageBubbleStyle();
+
+class MessageBubble extends StatelessWidget {
+  final MessageBubbleViewModel _viewModel;
+  final MessageBubbleStyle _style;
+  final Function _onTap;
+
+  MessageBubble({
+    @required MessageBubbleViewModel viewModel,
+    MessageBubbleStyle style,
```

Comment:
Add the defaultStyle here instead of adding below. You won't need to check if the incoming style is null or not
```suggestion
    MessageBubbleStyle style = _defaultStyle,
```